myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html

```diff
@@ -0,0 +1,50 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "request_email_lists_tmpl" %}
+
+
+{{#if show_simple }}
+<div class="">
+
+    <div>
+        <button>Create Mailing List</button>
+    </div>
+
+
+    <h3>Create Mailing List</h3>
+
+    <div>{{ course_name }}</div>
+    <div>{{ quarter }} {{ year }}</div>
+
+    <ul class="">
+        <li>Creates a single list for this section</li>
+        <li>Stays synced with the official class list</li>
+    </ul>
+
+    {{#if allow_advanced }}
+    <div><a href="">Advanced options</a></div>
+    {{/if}}
+</div>
+{{/if}}
+
+{{#if show_advanced }}
```

Comment:
@fanglinfang The "Advanced Options" link at the bottom of the form. https://invis.io/EQ9ZZ4KN7#/215841762_Email_-_Create_Form_-primary-